virttest/utils_test/qemu/__init__.py

```diff
@@ -462,6 +462,22 @@ def get_guest_free_mem(cls, vm):
         session = vm.wait_for_login(timeout=timeout)
```

Comment:
@aliang123  The dir seems wrong, can't be applied:

[root@hp-dl385g7-04 avocado-vt]# pwd
/home/staf-kvm-devel/workspace/avocado-vt
[root@hp-dl385g7-04 avocado-vt]# git am 753.patch
Applying: virttest:Add get_used_mem for windows guest balloon test
error: virttest/utils_test/qemu/__init__.py: does not exist in index